package.json

```diff
@@ -36,6 +36,8 @@
     "fast-glob": "^3.2.11",
     "fs-extra": "^10.0.0",
     "fuse.js": "^6.5.3",
+    "glob": "^7.1.6",
+    "markdown-it": "^12.0.4",
```

Comment:
These are required by docs-search. (Maybe at some point glob can be swapped out for fast-glob? They sound similar, I haven't investigated the differences.

Docs-search also had "puppeteer" in its package.json but a quick grep over the codebase indicated it was not in use.

Reply:
Yeah, would be great if we can use fast glob at some point